features/admins/admin_invitations.feature

```diff
@@ -256,7 +255,7 @@ Feature: Admin Actions to Manage Invitations
       And I should not see "Joining the Archive currently requires an invitation; however, we are not accepting new invitation requests at this time."
 
   Scenario: An admin can send an invitation to a user via email
-    Given I am logged in as superadmin
+    Given I am logged in as a "support" admin
```

Comment:
I think all the scenarios from here down are actually things that haven't been locked down yet. These things are done via admin/invitations, /invite_requests, and /invite_requests/manage, but we've only locked down pages like users/username/invitations. 

Ergo, per your description of the PR, we might want to use generic admin accounts here.

Reply:
Thanks, I'll switch to no-role admins. The only exception is

https://github.com/otwcode/otwarchive/blob/3b2cf664ea303b1066f7372b6930c9b51cef64b0/features/admins/admin_invitations.feature#L338

which does use /invitations in part, so I'll keep `as a "support" admin` there.